Public/Src/Cache/ContentStore/Distributed/Sessions/ReadOnlyDistributedContentSession.cs

```diff
@@ -964,6 +1030,30 @@ private class RemotePinning
                 BoolResult updated = await UpdateContentTrackerWithNewReplicaAsync(operationContext, new[] { new ContentHashWithSize(remote.ContentHash, copy.ContentSize) }, cancel, UrgencyHint.Nominal);
                 if (updated.Succeeded)
                 {
+                    if (Settings.ProactiveCopyOnPin && Settings.ProactiveCopyMode != ProactiveCopyMode.Disabled)

```

Comment:
Short comment here like "// Initiate a proactive copy if just pinned content is under-replicated"